std/v2alpha/algorithm/comparison.d

```diff
@@ -0,0 +1,24 @@
+module std.v2alpha.algorithm.comparison;
```

Comment:
I suggest `std2` instead of `std.v2`. `v2` should not be a sub-package of `std` version 1.

Reply:
Kinda makes sense. It would be a different story if we had this in the beginning, like if it already was std.v1 etc and now std.v2 etc.

General question arises (excuse me if this is already answered in the forums):
Would these be side by side?

Will you be able to selectively import stuff from std while still using std2?

import std;
import std2 : awesome;
--
import std2;
import std : awesome;

No conflict because the selective import always wins?